lib/mongo/cursor.ex

```diff
@@ -8,6 +8,13 @@ import Record, only: [defrecordp: 2]
 defmodule Mongo.Cursor do
   @moduledoc false
 
+  @type t :: %__MODULE__{
+    conn: Mongo.conn,
+    coll: Mongo.collection,
+    query: BSON.document,
+    select: BSON.document | nil,
+    opts: Keyword.t
+  }
```

Comment:
Perhaps I'm missing something, but that's already done [here](https://github.com/ericmj/mongodb/blob/master/lib/mongo.ex#L52)?